src/ui_utils.c

```diff
@@ -2566,6 +2582,10 @@ void ui_init(void)
 	ui_init_toolbar_widgets();
 	init_document_widgets();
 	create_config_files_menu();
+
+	// after UI is initialized, apply user's custom CSS (if it exists)
+	// to override other CSS styles
+	init_user_style();
```

Comment:
Originally I had it all done at the same time but it didn't work. I suspect we setup something in the intervening code that is required. If you're going to test this PR, just try moving `init_user_style` directly under `init_default_style` and it should be reproduced.